kratos/utilities/force_and_torque_utils.h

```diff
@@ -0,0 +1,119 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+
+#ifndef KRATOS_FORCE_AND_TORQUE_UTILS
+#define KRATOS_FORCE_AND_TORQUE_UTILS
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * 
+ */
+class KRATOS_API(KRATOS_CORE) ForceAndTorqueUtils
+{
+public:
+    ///@name Type Definitions
+    ///@{
+    
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /** Constructor.
+     */
+
+    /** Destructor.
+     */
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    static std::array<array_1d<double,3>,2> SumForceAndTorque(const ModelPart& rModelPart, const array_1d<double,3>& rReferencePoint);
```

Comment:
```suggestion
    static std::array<array_1d<double,3>,2> SumForceAndTorque(
        const ModelPart& rModelPart,
        const array_1d<double,3>& rReferencePoint);
```
Style guide.

Besides this, I think we should ask the user for the force variable.